src/it/MPIR-455/verify.groovy

```diff
@@ -0,0 +1,119 @@
+
+def summaryLine = '''\
+<tr class="b">
+<td align="left">module-info-only-test-0.0.1.jar</td>
+<td align="right">3.3 kB</td>
+<td align="right">-</td>
+<td align="right">-</td>
+<td align="right">-</td>
+<td align="center">-</td>
+<td align="center">-</td></tr>
+<tr class="a">
+<td align="left">&#160;&#160;&#160;&#x2022; Root</td>
+<td align="right">-</td>
+<td align="right">10</td>
+<td align="right">0</td>
+<td align="right">0</td>
+<td align="center">-</td>
+<td align="center">-</td></tr>
+<tr class="b">
+<td align="left">&#160;&#160;&#160;&#x2022; Versioned</td>
+<td align="right">-</td>
+<td align="right">1</td>
+<td align="right">1</td>
+<td align="right">1</td>
+<td align="center">11</td>
+<td align="center">No</td></tr>
+<tr class="a">
+<td align="left">multi-release-test-0.0.1.jar</td>
+<td align="right">10.2 kB</td>
+<td align="right">-</td>
+<td align="right">-</td>
+<td align="right">-</td>
+<td align="center">-</td>
+<td align="center">-</td></tr>
+<tr class="b">
+<td align="left">&#160;&#160;&#160;&#x2022; Root</td>
+<td align="right">-</td>
+<td align="right">37</td>
+<td align="right">1</td>
+<td align="right">1</td>
+<td align="center">1.8</td>
+<td align="center">Yes</td></tr>
+<tr class="a">
+<td align="left">&#160;&#160;&#160;&#x2022; Versioned</td>
+<td align="right">-</td>
+<td align="right">9</td>
+<td align="right">1</td>
+<td align="right">1</td>
+<td align="center">9</td>
+<td align="center">Yes</td></tr>
+<tr class="b">
+<td align="left">&#160;&#160;&#160;&#x2022; Versioned</td>
+<td align="right">-</td>
+<td align="right">9</td>
+<td align="right">1</td>
+<td align="right">1</td>
+<td align="center">11</td>
+<td align="center">Yes</td></tr>
+<tr class="a">
+<td align="left">plexus-utils-4.0.0.jar</td>
+<td align="right">192.4 kB</td>
+<td align="right">-</td>
+<td align="right">-</td>
+<td align="right">-</td>
+<td align="center">-</td>
+<td align="center">-</td></tr>
+<tr class="b">
+<td align="left">&#160;&#160;&#160;&#x2022; Root</td>
+<td align="right">-</td>
+<td align="right">128</td>
+<td align="right">86</td>
+<td align="right">7</td>
+<td align="center">1.8</td>
+<td align="center">Yes</td></tr>
+<tr class="a">
+<td align="left">&#160;&#160;&#160;&#x2022; Versioned</td>
+<td align="right">-</td>
+<td align="right">5</td>
+<td align="right">1</td>
+<td align="right">1</td>
+<td align="center">9</td>
+<td align="center">Yes</td></tr>
+<tr class="b">
+<td align="left">&#160;&#160;&#160;&#x2022; Versioned</td>
+<td align="right">-</td>
+<td align="right">5</td>
+<td align="right">1</td>
+<td align="right">1</td>
+<td align="center">10</td>
+<td align="center">Yes</td></tr>
+<tr class="a">
+<td align="left">&#160;&#160;&#160;&#x2022; Versioned</td>
+<td align="right">-</td>
+<td align="right">5</td>
+<td align="right">1</td>
+<td align="right">1</td>
+<td align="center">11</td>
+<td align="center">Yes</td></tr>'''
+
+assert html.contains( summaryLine.replaceAll( "\n", System.lineSeparator() ) )
```

Comment:
`summaryLine.normalize()` should do it as well. See https://github.com/apache/maven-reporting-impl/commit/610c5230c447102f9899a7ae35d30541c3b1df6c#diff-b244770c8a4089f904938a9ae65dec0b76fa696cf8b3f12bc8f6ae358cd00a55L30-L34